packages/rum-core/src/domain/contexts/userContext.spec.ts

```diff
@@ -48,13 +35,68 @@ describe('user context', () => {
       name: '2',
     })
   })
+
+  describe('assemble hook', () => {
+    it('should set the user', () => {
+      userContext.setContext({ id: '123', foo: 'bar' })
+      const event = hooks.triggerHook(HookNames.Assemble, { eventType: 'view', startTime: 0 as RelativeTime })
+
+      expect(event).toEqual({
+        type: 'view',
+        usr: {
+          id: '123',
+          foo: 'bar',
+        },
+      })
+    })
+
+    it('should set the user.anonymous_id when trackAnonymousUser is true', () => {
```

Comment:
Ultimate nit-picky nit:
```suggestion
    it('should set usr.anonymous_id when trackAnonymousUser is true', () => {
```

(Or perhaps just `should set anonymous_id...`.)